src/unix/linux.c

```diff
@@ -2281,6 +2281,131 @@ uint64_t uv_get_available_memory(void) {
 }
 
 
+#define BUF_SIZE 1024
+
+static uv__cpu_constraint uv__get_cgroupv2_constrained_cpu(const char *cgroup) {
```

Comment:
Style: star leans left
```suggestion
static uv__cpu_constraint uv__get_cgroupv2_constrained_cpu(const char* cgroup) {
```